src/test/java/seedu/guest/model/UserPrefsTest.java

```diff
@@ -13,9 +21,59 @@ public void setGuiSettings_nullGuiSettings_throwsNullPointerException() {
     }
 
     @Test
-    public void setAddressBookFilePath_nullPath_throwsNullPointerException() {
+    public void setGuestBookFilePath_nullPath_throwsNullPointerException() {
         UserPrefs userPrefs = new UserPrefs();
         assertThrows(NullPointerException.class, () -> userPrefs.setGuestBookFilePath(null));
     }
 
+    @Test
+    public void getGuiSettings_success() {
+        UserPrefs userPrefs = new UserPrefs();
+        assertEquals(userPrefs.getGuiSettings(), new UserPrefs().getGuiSettings());
+    }
+
+    @Test
+    public void getGuestBookFilePath_success() {
+        UserPrefs userPrefs = new UserPrefs();
+        assertEquals(userPrefs.getGuestBookFilePath(), new UserPrefs().getGuestBookFilePath());
+    }
+
+    @Test
+    public void equals() {
+        UserPrefs userPref = new UserPrefs();
+
+        // same object -> returns true
+        assertTrue(userPref.equals(userPref));
+
+        // null -> returns false
+        assertFalse(userPref.equals(null));
+
+        // different GuiSettings -> returns false
+        UserPrefs editedUserPref = new UserPrefs();
+        editedUserPref.setGuiSettings(new GuiSettings(1.0, 1.0, 1, 1));
+        assertFalse(userPref.equals(editedUserPref));
+
+        // different GuestBookFilePath -> returns false
+        editedUserPref = new UserPrefs();
+        editedUserPref.setGuestBookFilePath(Paths.get("address/book/file/path"));
+        assertFalse(userPref.equals(editedUserPref));
+    }
+
+    @Test
+    public void hashCodeTest() {
```

Comment:
Should we standardise the naming between `hashcode()` and `hashCodeTest()`?

Reply:
Agreed. Changed to hashcode().